rusoto/services/s3/src/custom/util.rs

```diff
@@ -0,0 +1,19 @@
+use rusoto_core::signature;
+
+/// URL encodes an S3 object key. This is necessary for `copy_object` and `upload_part_copy`,
+/// which require the `copy_source` field to be URL encoded.
+///
+/// # Examples
+///
+/// ```
```

Comment:
Oops, good catch! I had them in there originally, must have accidentally deleted it. Though it still rendered fine and passed doctests.